### PR TITLE
feat(README): add Discourse Archives list

### PR DESCRIPTION
**Pull request by meowsbits**
Rel #235 

:rabbit2: :tornado:  https://github.com/meowsbits/ECIPs/commit/7c3048010e8b609d30e634b9c2bb601f2a185970

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ When you believe your ECIP is mature and ready to progress past the draft phase,
 
  - **For a Standards Track ECIP of type Core**, ask to have your issue added to the agenda of an upcoming core developer team meeting. There, it can be discussed for inclusion in a future hard fork. If developers agree to include it, the ECIP editors will update the state of your ECIP to 'Accepted'.
  - **For all other ECIPs**, open a PR changing the state of your ECIP to 'Final'. An editor will review your draft and ask if anyone objects to its finalization. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the ECIP - they may close the PR and request that you fix the issues in the draft before trying again.
-
+
```
Comment (meowsbits) [inline README.md]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -70,6 +70,15 @@ Every pull request will be reviewed and discussed by core and volunteer Ethereum
 
 Having an ECIP within the folder of the repository does not make it a formally accepted standard until its status becomes "Final". For an ECIP to become Final it requires the common consent of the ecosystem. Those proposing changes should consider that, ultimately, consent may rest with the consensus of the Ethereum Classic implementers and users.
 
+## Disourse Archives
```
Comment (meowsbits) [inline README.md]:
```suggestion
## Discourse Archives
```

**Comment by meowsbits** [comment]
:rabbit2: :tornado: :tornado:  https://github.com/ethereumclassic/ECIPs/pull/241#issue-348876531 https://github.com/meowsbits/ECIPs/commit/d13d493321f1dd34fb90716362df8c562a7cb3fc

**Comment by meowsbits** [comment]
:tornado: ^nth https://github.com/meowsbits/ECIPs/commit/b8ba51fd29803ece462e42cee3869b9a2194421e

**Comment by BelfordZ** [comment]
This is really great, thanks @meowsbits 

One question, is there a way that the code for this can be integrated with the repo directly? for example, perhaps this can be run periodically, or on merge to master, and put the contents into release artifacts or something?

**Comment by meowsbits** [comment]
@BelfordZ Great question! Let's open a new Issue to figure something out...

**Comment by BelfordZ** [comment]
LGTM @soc1c

**Pull request merged**